backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/image/ImagesHandler.java

```diff
@@ -1065,6 +1065,22 @@ public Set<Guid> getVolumeChain(Guid vmId, Guid vdsId, DiskImage activeImage) {
         return images;
     }
 
+    public boolean isSnapshotUsed(VM vm, DiskImage image) {
+        if (vm.isRunningOrPaused()) {
+            Set<Guid> volumeChain = getVolumeChain(vm.getId(),
```

Comment:
full-list is a relatively costly call, would be better to call it once and not once-per-disk but as we don't expect the number of disks to be so high, it's ok to consider that separately

Reply:
makes sense, I'll set up an issue to follow up here